api/python/provisioner/srv/salt/firewall/config.sls

```diff
@@ -25,6 +25,7 @@ saltmaster:
     - ports:
       - 4505/tcp
       - 4506/tcp
+      - 8301/tcp
```

Comment:
Consul is supposed to run on private data interface. Comment this entry for now.